Makefile

```diff
@@ -42,11 +44,23 @@ help:
 
 # Install python deps via pip
 deps:
-	$(PIP) install -r requirements.txt
+	$(PIP_INSTALL) -r requirements.txt
 
 # Install
-install:
-	$(PIP) install .
+install: patch-pix2pixhd
+	$(PIP_INSTALL) .
+.PHONY: patch-pix2pixhd
+
+# Patch pix2pixhd to trick it into thinking it was part of this mess
+PIX2PIX_FILES = ocrd_anybaseocr/pix2pixhd/*/*.py ocrd_anybaseocr/pix2pixhd/*.py
+patch-pix2pixhd: pix2pixhd
+	sed -i 's,^from util,from ..util,' $(PIX2PIX_FILES)
+	sed -i 's,^import util,import ..util,' $(PIX2PIX_FILES)
+	# string exceptions, srsly y
+	sed -i "s,raise('\([^']*\)',raise(Exception('\1')," $(PIX2PIX_FILES)
```

Comment:
It does have a `__init__.py` which gave me the idea to use as a subpackage in the first place. Replacing the import stmts cannot be avoided because they assume pix2pixhd to be the root package. The code is using string exceptions which are a hard nope for python 3